### ISSUE TITLE
1.8 issues

### ISSUE THREAD

**Issue by stefanpenner** [testem/testem#886]
As for 1.8 I get :

```sh
± % ember test
Built project successfully. Stored in "/Users/stefanpenner/src/y/dsp/tmp/class-tests_dist-UPWBY9zr.tmp".
not ok 1 unknown error
    ---
        message: >
            undefined
    ...

1..1
# tests 1
# pass  0
# skip  0
# fail  1
127
```

https://github.com/testem/testem/compare/v1.7.4...v1.8.0

still debugging.

**Comment by stefanpenner** [comment]
looks to be  9e4e14e006e3ab7d1ed1b23f48fa5107402652c3

**Comment by johanneswuerbach** [comment]
Could you share your testem config? 127 looks like phantomjs isn't able to find the launch script.

**Comment by stefanpenner** [comment]
```
{
  "framework": "qunit",
  "test_page": "tests/index.html?coverage=true&hidepassed&dockcontainer",
  "disable_watching": true,
  "launch_in_ci": [
    "PhantomJS"
  ],
  "launch_in_dev": [
    "PhantomJS",
    "Chrome"
  ]
}
```

**Comment by johanneswuerbach** [comment]
Could you share the debug output of testem? You can get it by adding `debug: true,` here https://github.com/ember-cli/ember-cli/blob/master/lib/tasks/test.js#L49 and then looking for a `testem.log` in your project directory.

**Comment by johanneswuerbach** [comment]
Ping, still can't reproduce this. @chadhietala you saw it on travis?

**Comment by johanneswuerbach** [comment]
Ah looks like https://travis-ci.org/tildeio/glimmer/builds/134823530 had the same issue and the glimmer testem yml us doing some phantom PATH dancing, will investigate.

**Comment by johanneswuerbach** [comment]
@stefanpenner what version of phantomjs are you using? also 2.0?

**Comment by johanneswuerbach** [comment]
Fix published in `v1.8.1`
